api_tests/lib/actor.ts

```diff
@@ -111,6 +112,14 @@ export class Actor {
             case "bitcoin-broadcast-signed-transaction": {
                 let { hex } = action.payload;
 
+                let expiry = parseInt(action.payload.invalid_until);
```

Comment:
:thinking: Does this even work? Shouldn't it be `action.invalid_until`?

Reply:
yes well spotted. We now do the waiting in the actual test framework.